ui_framework/components/local_nav/_local_search.scss

```diff
@@ -44,6 +48,15 @@
     transform: translateY(-50%); /* 1 */
   }
 
+  .kuiLocalSearchAssistedInput__assistanceLink {
+    display: none;
+  }
+
+  .kuiLocalSearchInput:focus + .kuiLocalSearchAssistedInput__assistance .kuiLocalSearchAssistedInput__assistanceLink,
+  .kuiLocalSearchAssistedInput__assistanceLink:active {
+    display: inherit;
+  }
+
```

Comment:
It seems like we have that problem anyway:

![lucene_help_link](https://cloud.githubusercontent.com/assets/1238659/25717822/9b1ba4be-30b8-11e7-98f3-1878ebf956f8.gif)

As long as we hide that link by default, it's going to create a strange interaction when users try to click the text that it invisibly occupies. Maybe we should go back to showing it all the time? @snide what do you think?